src/main/java/ch/njol/skript/conditions/CondIsSkriptCommand.java

```diff
@@ -18,7 +18,7 @@
  */
 package ch.njol.skript.conditions;
 
-import static ch.njol.skript.command.Commands.skriptCommandExists;
+import static ch.njol.skript.command.Commands.scriptCommandExists;
```

Comment:
I'm not sure there's a reason for this to be a static import considering it's used only once. We may want to consider changing it while we're here :)